From 3b46bb2a4eada7f29e016d586229782f6d1c35bb Mon Sep 17 00:00:00 2001
From: Pino Toscano <toscano.pino@tiscali.it>
Date: Sat, 9 Jan 2016 11:47:37 +0100
Subject: [PATCH] cmake: fix DESTDIR installation for python

When running `setup.py install`, respect the value of the DESTDIR
environment variable so staged installs (used by distros, for example)
work.
---
 apps/CMakeLists.txt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/apps/CMakeLists.txt b/apps/CMakeLists.txt
index 70c77fd..2f6f8f1 100644
--- a/apps/CMakeLists.txt
+++ b/apps/CMakeLists.txt
@@ -42,5 +42,5 @@ IF(PYTHON)
     IF(NOT DEFINED SETUP_PY_INSTALL_PREFIX)
       SET(SETUP_PY_INSTALL_PREFIX "${CMAKE_INSTALL_PREFIX}")
     ENDIF()
-    INSTALL(CODE "execute_process(COMMAND ${PYTHON} ${SETUP_PY} install --prefix=${SETUP_PY_INSTALL_PREFIX})")
+    INSTALL(CODE "execute_process(COMMAND ${PYTHON} ${SETUP_PY} install --prefix=\$ENV{DESTDIR}${SETUP_PY_INSTALL_PREFIX})")
 ENDIF(PYTHON)
-- 
2.6.4


